meta/documents/user_guide/de/initial_setup.md

```diff
@@ -0,0 +1,187 @@
+
+7. Klicken Sie in der Symbolleiste über den Einstellungen auf die Schaltfläche `Speichern`. ![save button](../../../images/common/button_save.png)
+
+## 2. Aktivieren der Auftragsherkunft
+Die Auftragsherkunft in Plentymarkets identifiziert den Vertriebskanal, in dem ein Auftrag generiert wurde. Damit das Plentymarkets-System Aufträge ordnungsgemäß von der Wayfair-API importieren kann, muss die Wayfair-Auftragsherkunft aktiviert sein:
+
+1. Gehen Sie auf der Hauptseite von Plentymarkets zu `Einrichtung` >>` Aufträge` >> `Auftragsherkunft`.
+
+    ![order referrer menu](../../../images/de/menu_order_referrer.png)
+
+2. Setzen Sie ein Häkchen neben den Bestellempfänger `Wayfair`.
+
+    ![wayfair order referrer](../../../images/common/wayfair_referrer_checked.png)
+
+3. Klicken Sie auf `Speichern` ![save button](../../../images/common/button_save.png).
+
+## 3. Einrichten von Plentymarkets für den Versand über Wayfair
+Befolgen Sie die [Anweisungen für den Versand über Wayfair](wayfair_shipping.md) beschriebenen Verfahren, um eine ordnungsgemäße Integration mit Wayfair für den Versand von Bestellartikeln sicherzustellen.
+
+## 4. Verknüpfen Sie Wayfair-Bestellartikel mit Plentymarkets-Artikeln:
+Um eingehende Bestellungen von Wayfair ordnungsgemäß verarbeiten zu können, muss das Wayfair-Plugin die Lieferanten-Teilenummern aus Wayfairs Systemen mit einem bestimmten Feld von Artikelvariationen in Plentymarkets abgleichen. Standardmäßig geht das Wayfair-Plugin davon aus, dass die `Variationsnummer` **(nicht die ID der Varianten)** der Varianten eines Artikels in Plentymarkets mit der Teilenummer des Wayfair-Lieferanten übereinstimmt.
+
+![variation number field](../../../images/de/variation_number_field.png)
+
+Wenn die Lieferanten-Teilenummern für Ihre Organisation in einem alternativen Feld in Ihren Plentymarkets-ArtikelVarianten angezeigt werden, ändern Sie den Wert von [`Item Mapping Method`](settings_guide.md#item-mapping-method).
+
+## 5. Artikel auf Wayfair zum Verkauf anbieten
+Artikel, die Sie auf Wayfair verkaufen möchten, müssen in Plentymarkets als aktiv markiert werden. Der Benutzer von Plentymarkets kann auch festlegen, welche Artikel auf Wayfair zum Verkauf stehen. **Beachten Sie, dass der Lagerbestand und die bestellten Artikel auf der Ebene "Varianten" gesteuert werden.**
+
+Dieses Verfahren ist nur erforderlich, wenn [die Einstellung `Vollständigen Bestand an Wayfair senden?`](settings_guide.md#vollständigen-bestand-an-wayfair-senden) **deaktiviert** ist:
+
+1. Gehen Sie auf der Hauptseite von Plentymarkets zu `Artikel` >> `Artikel bearbeiten`.
+
+2. Suchen Sie nach Artikeln und öffnen Sie sie
+
+3. **Klicken Sie für jedes Element** auf `Varianten` und öffnen Sie diese.
+
+4. **Für jede Variante**:
+
+    1. Stellen Sie auf der Registerkarte `Einstellungen` sicher, dass das Kontrollkästchen `Aktiv` im Abschnitt `Verfügbarkeit` aktiviert ist.
+
+        ![variation active](../../../images/de/variation_active_field.png)

```

Comment:
This screenshot does not have the Zwiesitzer Amsterdam at Dawn blurred like in the English screenshot